### DISCUSSION TITLE
swarmbytes Support?

### DISCUSSION THREAD

**Discussion by Cr4ckPad** [engageub/InternetIncome#15] [Q&A]
Is the application compatible with the SwarmBytes Support website @engageub

**Comment by engageub** [comment]
This has not been tested with swarmbytes since it requires 250 IPS to sign up and download of the application seems to be available only for registered users.


Thank you

**Comment by engageub** [comment]
Thank you for the info. Yes, may I know from where you purchased IP's. Are you an ISP provider.

**Comment by engageub** [comment]
Hello,
Could you please let me know the monthly estimated income from these IPs. Is it more than honeygain. 
I am planning to purchase IPs and provide them but would like to know if its profitable.

Thank you

**Reply by engageub** [reply]
Hi,
Thank you for the response. I do not require the script as I need IPs and also need to get approval from swarmbytes.
I just wanted to know how much they pay per IP when using swarmbytes.

Thank you

**Reply by engageub** [reply]
Thank you for the detailed response. 
I was considering purchasing the IPs at a rate of approximately $0.50 per IP. However, after calculating the projected revenue, it appears that this would result in a loss for me.
Honeygain is more profitable when compared to Swarmbytes with respect to IP revenue. 

Thank you

**Reply by superit63** [reply]
WOW. how can you purchase the IPs with just 0.5$/IP. Its amazing if you can share the provider. Thanks

**Reply by engageub** [reply]
Well, the provider cannot guarantee only residential proxies at the moment. There are some issues ongoing and I will let you know when they are working good. 

Thank you
